src/calculateRentalCost.js

```diff
@@ -1,10 +1,19 @@
 /**
- * @param {number} days
- *
- * @return {number}
+ * Calculates the rental cost based on the number of days.
+ * @param {number} days - The number of days the car is rented.
+ * @return {number} - The total rental cost.
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
```

Comment:
Since 'totalCost' is not reassigned, you can use 'const' instead of 'let' to declare it. This will prevent unintentional changes.